clang/lib/Sema/SemaRISCV.cpp

```diff
@@ -1,12 +1,12 @@
-//==- SemaRISCVVectorLookup.cpp - Name Lookup for RISC-V Vector Intrinsic -==//
+//==------- SemaRISCV.cpp ------- RISC-V target-specific routines ----------==//
```

Comment:
The prevailing style starts with `//===-` or `//===--`

Reply:
I never noticed this. Thanks!